packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -337,9 +344,38 @@ import { name, version } from '../package'
             choices: ['TypeScript', 'JavaScript'],
             message: 'Select your preferred coding language',
             initial: 'TypeScript',
+            onCancel: () => process.exit(1),
           })
-
           task.output = ctx.language
+          // Error code and exit if someone has disabled yarn install but selected JavaScript
+          if (yarnInstall === false && ctx.language === 'JavaScript') {
+            throw new Error(
+              'JavaScript transpilation requires running yarn install. Please rerun Create-Redwood-App without disabling yarn install.'
+            )
+          }
+        },
+        options: {
+          persistentOutput: true,
+        },
+      },
+      {
+        title: 'Git preference',
+        skip: () => gitInit !== null,
+        task: async (ctx, task) => {
+          ctx.gitInit = await task.prompt({
+            type: 'Toggle',
+            message: 'Do you want to initialize a new git repo?',
+            enabled: 'Yes',
+            disabled: 'No',
```

Comment:
A common thing CLIs like lerna do is have a prompt like `Yn` or `Y\n`—the "Y" is capitalized to let users know it's the default. Even though we send focus to `Yes`, maybe it's worth mimicking the style?

```suggestion
            disabled: 'no',
```

Reply:
I like this idea - that was one of my reservations about the toggle. I didn't love the way the Yes was highlighted on the right side.  Didn't feel quite right to me.